jsonnet/jsonnetfile.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": 1,
+  "dependencies": [
+    {
+      "source": {
+        "local": {
+          "directory": "lib"
+        }
+      },
+      "version": "."
+    }
```

Comment:
Definitely remove this local dependency, but probably we don't need this file anymore to begin with.